src/components/modal-with-close-button.tsx

```diff
@@ -78,7 +78,7 @@ export function ModalWithCloseButton({
             onClick={onRequestClose}
             title={strings.share.close}
             dataQa="modal-close-button"
-            className={closeButtonClassName}
+            className={clsx('absolute right-2 top-2', closeButtonClassName)}
```

Comment:
```suggestion
            className={clsx('absolute right-3.5 top-3.5', closeButtonClassName)}
```
that was the old value